.gitignore

```diff
@@ -0,0 +1 @@
+/.idea
```

Comment:
I don't think we should add this file with this pull request. You can have that file locally and ignore itself even.